ska_conda/pkg_defs/Chandra.Maneuver/meta.yaml

```diff
@@ -30,12 +31,15 @@ requirements:
     - python
     - setuptools
     - six
-    - testr
     - quaternion
     - chandra.time
     - ska.sun
 
 test:
+  requires:
+    - agasc
```

Comment:
This is a case where we have some involved testing that seems to need agasc and Ska.engarchive.  I suppose it would be fine to just add those as runtime dependencies, but it seemed overkill (they don't really seem like run-time requirements for the package even if they are needed to complete all of its tests).

Reply:
Interesting web we are weaving.  Agreed this is fine to leave as is.